tests/components/xiaomi_ble/test_event.py

```diff
@@ -174,7 +224,8 @@ async def test_events(
         state = hass.states.get(meas["entity"])
         attributes = state.attributes
         assert attributes[ATTR_FRIENDLY_NAME] == meas[ATTR_FRIENDLY_NAME]
-        assert attributes[ATTR_EVENT_TYPE] == meas[ATTR_EVENT_TYPE]
+        if ATTR_EVENT_TYPE in attributes:
+            assert attributes[ATTR_EVENT_TYPE] == meas[ATTR_EVENT_TYPE]
```

Comment:
We try to avoid branching in tests. Usually it means splitting the test or adjusting the params to avoid the need to branch

Reply:
I've split the tests of the fingerprint and lock. These advertisements not only contain event data, but also sensor and binary sensor data.